src/ApiTestCase.php

```diff
@@ -154,6 +170,33 @@ protected function getJsonResponseFixture($filename)
         )), true);
     }
 
+    protected function loadPlatformData()
+    {
+        $loader = new Loader();
+
+        $directory = $this->getFixturesFolder();
+        $files = $this->getFilesFromDirectory($directory);
+
+        foreach ($files as $file) {
+            $objects = $loader->load($directory.$file);
+            $this->persistObjects($objects);
+        }
+
+        $this->entityManager->flush();
+    }
+
+    private function persistObjects($objects)
+    {
+        foreach ($objects as $object) {
+            $this->entityManager->persist($object);
+        }
+    }
+
+    private function getFixturesFolder()
+    {
+        return $this->getRootDir().'/../src/DataFixtures/ORM/';
```

Comment:
We should make it configurable with `/../src/DataFixtures/ORM/` as a default value.